topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -1685,7 +1699,9 @@ Galaxy is now configured with an admin user, a database, and a place to store da
 >               - workflow-scheduler
 >    +
 >    +# systemd
->    +galaxy_manage_systemd: yes
+>    +galaxy_manage_systemd: true
+>    +galaxy_manage_gravity: "{{ false if __galaxy_major_version is version('22.01', '<') else true }}"
+>    +galaxy_systemd_mode: "{{ 'mule' if __galaxy_major_version is version('22.01', '<') else 'gravity' }}"
```

Comment:
This shouldn't be user controlled, we don't want that. It should be in the role.

Reply:
the role currently default to 22.05. Should I pr the role then?

Reply:
hm I missed that it was 22.01 here, we want to use gravity starting with 22.05, not if it's <22.01. The playbook is right, any reason to override that here? You want to deploy 22.01 with mules/gravity?

I'm not sure we want to teach that in the training materials, I think these should be left default, and we're going to teach 22.05 this summer.